packages/components/package.json

```diff
@@ -30,10 +30,11 @@
 	],
 	"dependencies": {
 		"@babel/runtime": "^7.16.0",
-		"@emotion/cache": "^11.4.0",
-		"@emotion/css": "^11.1.3",
-		"@emotion/react": "^11.4.1",
-		"@emotion/styled": "^11.3.0",
+		"@emotion/cache": "^11.7.1",
+		"@emotion/css": "^11.7.1",
+		"@emotion/react": "^11.7.1",
+		"@emotion/serialize": "^1.0.2",
```

Comment:
Discovered by @youknowriad in #37324. This dependency for types was missing. It worked only because it's a dependency of dependencies.

Reply:
Good find, thank you!